.travis.yml

```diff
@@ -21,6 +21,7 @@ branches:
 
 solution: src/Cassandra.sln
 install:
+  - ./build/scassandra_install.sh
```

Comment:
No need to install scassandra on Travis: We use scassandra for integration tests, which are not run on Travis.